### PR TITLE
make dnsNames configurable for dex and kube-oidc-proxy

### PR DESCRIPTION
**Pull request by joejulian**
**What type of PR is this?**

bug

**What this PR does/ why we need it**:

need to be able to specify a different hostname for kommander 2
**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
none
```

**Checklist**

* [ ] *If a chart is changed, the chart version is correctly incremented.*
* [ ] The commit message explains the changes and why are needed.
* [ ] The code builds and passes lint/style checks locally.
* [ ] The relevant subset of integration tests pass locally.
* [ ] The core changes are covered by tests.
* [ ] The documentation is updated where needed.

### CODE REVIEW

File: staging/traefik-forward-auth/Chart.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ apiVersion: v1
 appVersion: "latest"
 description: Minimal forward authentication service that provides OIDC based login and authentication for the traefik reverse proxy
 name: traefik-forward-auth
-version: 0.2.14
+version: 0.2.15
```
Comment (gracedo) [inline staging/traefik-forward-auth/Chart.yaml:6]: maybe you meant to bump dex chart version instead of tfa? (I don't see any tfa chart changes)

Comment (joejulian) [reply staging/traefik-forward-auth/Chart.yaml]: nope! thanks.

File: stable/dex/Chart.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: dex
-version: 2.9.3
+version: 2.9.5
```
Comment (jr0d) [inline stable/dex/Chart.yaml:3]: 4?

**Review by jr0d — APPROVED** [review]

File: stable/dex/Chart.yaml (hunk 2 above)
Comment (gracedo) [inline stable/dex/Chart.yaml:3]:
```suggestion
version: 2.9.4
```

**Review by gracedo — APPROVED** [review]

**Pull request merged**